main/level_1.tscn

```diff
@@ -15,3 +15,9 @@ script = ExtResource("1_d3jfb")
 
 [node name="PauseMenu" parent="." instance=ExtResource("3_sns4i")]
 process_mode = 3
+
+[node name="Timer" type="Timer" parent="."]
+wait_time = 2.0
```

Comment:
Fight

Reply:
I think this will be a bit hard to reach a conclusion of what the timer should be in the start since its hard to visualize te, like the collision issue above i believe this should be discussed in the next meeting because this value will change throughout the "rounds/waves" of enemies and that should be defined, so perhaps the solution would be in a later date a ticket be open to "solve" this value if necessary

Reply:
Yap, I agree, for me, it's ok, but let's wait for someone else say something :)